services/ingestor/src/main/kotlin/company/thewebhook/ingestor/Application.kt

```diff
@@ -1,15 +1,114 @@
 package company.thewebhook.ingestor
 
+import company.thewebhook.eventstore.MessageTooLargeException
+import company.thewebhook.eventstore.Producer
 import company.thewebhook.ingestor.plugins.*
+import io.ktor.http.*
 import io.ktor.server.application.*
+import io.ktor.server.config.*
 import io.ktor.server.netty.*
+import io.ktor.server.plugins.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import io.ktor.util.*
+import kotlinx.coroutines.launch
+import kotlinx.datetime.Clock
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.cbor.Cbor
+import kotlinx.serialization.encodeToByteArray
+import org.koin.ktor.ext.inject
 
 fun main(args: Array<String>): Unit = EngineMain.main(args)
 
-fun Application.module() {
+@Serializable
+data class WebhookRequestData(
```

Comment:
Isn't the standard to declare these under 'models' package

Reply:
Ye true

Reply:
Ah yes. Just didn't wanna overengineer a v simple module, but it makes sense to do it for the sake of consistency.